pod/perlgit.pod

```diff
@@ -22,12 +22,9 @@ I<github.com>.
 
 You can make a read-only clone of the repository by running:
 
-  % git clone git://github.com/Perl/perl5.git perl
+  % git clone git@github.com:Perl/perl5.git perl
```

Comment:
A clone made over ssh is not necessarily read-only, it depends on your user's permissions on the repo. Also, this only works for users with ssh keys registered on github, so it might be better to give the HTTPS example first, since that works anonymously.

OTOH, the bits about creating a fork and a pull request further down do assume the user has a github account, so that might be unnecessary.

Reply:
I agree, I use HTTPS for all read-only clones. This URL would be https://github.com/Perl/perl5.git

Reply:
This seems to have been reverted to the original SSH clone change?

Reply:
If intentional, some additional explanation would be appropriate: it is not a read-only clone, and it requires you to set up SSH keys for github, and there are no "firewall reasons" relevant to it or the HTTPS clone option.